tst/EnergyPlus/unit/CurveManager.unit.cc

```diff
@@ -76,13 +75,575 @@ TEST_F(EnergyPlusFixture, CurveExponentialSkewNormal_MaximumCurveOutputTest)
         "  1.;                      !- Maximum Curve Output",
     });
 
+    //documentation input output reference should tell me how to plot this equation
```

Comment:
Leftover comment? 

Also, see https://github.com/NREL/EnergyPlus/pull/7666#issuecomment-581361460

Reply:
Yes, this is a leftover commend. Good catch, thank you!